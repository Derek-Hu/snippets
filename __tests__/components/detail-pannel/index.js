import React from 'react';
import DetailPannel from '~/components/detail-pannel/index';

it('Tab样式', () => {
  const wrapper = mount(<DetailPannel />);

  expect(wrapper).toMatchSnapshot();
});
