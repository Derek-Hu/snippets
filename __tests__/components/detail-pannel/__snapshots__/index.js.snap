// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab样式 1`] = `
<_default>
  <section
    className="contentSection"
  >
    <div
      className="contentSectionHeader"
    />
    <div
      className="contentSectionMain"
    />
  </section>
</_default>
`;
